client/js/render.js

```diff
@@ -8,6 +8,7 @@ const utils = require("./utils");
 const sorting = require("./sorting");
 const constants = require("./constants");
 const condensed = require("./condensed");
+const helpers_parse = require("./libs/handlebars/parse");
```

Comment:
Underscore case oO
Just `parse` instead?

Reply:
This matches how we do it in other files.